File: expense-tracker-backend/src/main/java/matthias/expense_tracker/ExpenseTrackerApplication.java
package matthias.expense_tracker;

import matthias.expense_tracker.common.jpa.BaseRepositoryImpl;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@EnableJpaAuditing
@EnableTransactionManagement
@EnableJpaRepositories(repositoryBaseClass = BaseRepositoryImpl.class)
@SpringBootApplication
public class ExpenseTrackerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ExpenseTrackerApplication.class, args);
    }

}

File: expense-tracker-backend/src/main/kotlin/matthias/expense_tracker/ExpenseTrackerApplication.kt
package matthias.expense_tracker

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.PropertySource

@PropertySource("classpath:/liquibase.properties")
@SpringBootApplication
class ExpenseTrackerApplication

fun main(args: Array<String>) {
    runApplication<ExpenseTrackerApplication>(*args)
}
